data/preferences/src/components/about-page/ulauncher.svg.html

```diff
@@ -0,0 +1 @@
+../../../../media/icons/hicolor/ulauncher.svg
```

Comment:
It's a symlink. `ng-include` didn't work with `.svg` files, so I did this trick with `.svg.html`